cirq-ft/cirq_ft/infra/gate_with_registers.py

```diff
@@ -51,6 +51,60 @@ def __repr__(self):
         return f'cirq_ft.Register(name="{self.name}", shape={self.shape})'
 
 
+def total_bits(registers: Iterable[Register]) -> int:
```

Comment:
I know none of these had docstrings, but it might be a good time to add a quick one-line docstring to each

Reply:
added one line docstrings